test.c

```diff
@@ -0,0 +1,754 @@
+/* This file was automatically generated by CasADi.
```

Comment:
remove this file?

Reply:
It appears every time I run the tests. Definitely will remove it when this submodel is finished.